src/plugins/data_source_management/server/plugin.ts

```diff
@@ -0,0 +1,43 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
```

Comment:
License header looks different compare to other TS file.

Reply:
This is the header generated by the `node scripts/precommit_hook --fix` -- therefore I think this header should be sufficient (least possible). Let me know if this header need to be consistent.

Reply:
Good to learn, thanks

Reply:
Not sure this script is updated. I would suggest follow the best practices mentioned here https://github.com/opensearch-project/OpenSearch-Dashboards/blob/main/.eslintrc.js#L31-L55